modules/500-upmeter/images/upmeter/pkg/probe/checker/k8s_pod.go

```diff
@@ -107,7 +107,7 @@ func (c *podLifecycleChecker) new(pod *v1.Pod) check.Checker {
 
 	collectGarbage := newGarbageCollectorCheckerByLabels(c.access, pod.Kind, c.namespace, pod.Labels, c.garbageCollectionTimeout)
 
-	timeout := c.creationTimeout + c.schedulingTimeout + c.deletionTimeout
+	timeout := controlPlaneChecker.Timeout + c.garbageCollectionTimeout + c.creationTimeout + c.schedulingTimeout + c.deletionTimeout
```

Comment:
I'm not sure if control plane checker and garbage collection timeouts were left out on purpose or not

Reply:
Ok, I've reverted the change back